pkg/app/piped/cmd/piped/piped.go

```diff
@@ -237,18 +237,19 @@ func (p *piped) run(ctx context.Context, input cli.Input) (runErr error) {
 		return err
 	}
 
+	// if control plane's flag monitoring.enabled is false, otel provider logs errors.
+	// it's no problem but we don't want to see it.
+	// so we discard the errors and the logs.
+	otel.SetErrorHandler(otel.ErrorHandlerFunc(func(err error) {}))
+	otel.SetLogger(logr.Discard())
```

Comment:
we have to set global ErrorHandler and Logger before we create the tracer providers.